pkgs/development/libraries/xxHash/default.nix

```diff
@@ -11,21 +17,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-2WoYCO6QRHWrbGP2mK04/sLNTyQLOuL3urVktilAwMA=";
   };
 
-  # Upstream Makefile does not anticipate that user may not want to
-  # build .so library.
-  postPatch = lib.optionalString stdenv.hostPlatform.isStatic ''
-    sed -i 's/lib: libxxhash.a libxxhash/lib: libxxhash.a/' Makefile
-    sed -i '/LIBXXH) $(DESTDIR/ d' Makefile
-  '';
+  patches = [ ./man_location.patch ];
 
-  outputs = [ "out" "dev" ];
+  nativeBuildInputs = [ cmake ];
 
-  makeFlags = [ "PREFIX=$(dev)" "EXEC_PREFIX=$(out)" ];
+  cmakeDir = "../cmake_unofficial";
 
-  # pkgs/build-support/setup-hooks/compress-man-pages.sh hook fails
-  # to compress symlinked manpages. Avoid compressing manpages until
-  # it's fixed.
-  dontGzipMan = true;
+  cmakeFlags = [
+    "-DXXHASH_BUILD_XXHSUM=${if buildXXHSum then "ON" else "OFF"}"
```

Comment:
Is there a reason not to build this? It could go to `bin` output.

Reply:
I don't think we need that option just because it exists in cmake.

Reply:
Alright, it's dead